pkg/sense_hat_adapter.py

```diff
@@ -245,13 +254,28 @@ def update(self):
             self.device.notify_property_changed(self)
 
     def set_value(self, value):
+        print("info: sense_hat." + self.name + " from " + str(self.value) + " to " + str(value))
         if self.name == 'message':
             self.device.controller.show_message(value)
         elif self.name == 'color':
-            color = [int(value[1:3], 0x10),
-                     int(value[3:5], 0x10),
-                     int(value[5:7], 0x10)]
-            self.device.controller.clear(color)
+            if self.device.properties['lightOn'].value:
+                color = [int(value[1:3], 0x10),
+                         int(value[3:5], 0x10),
```

Comment:
I will update color event if off

Reply:
It updates the color at line 279.  But it doesn't turn on the light if it is off.

Reply:
There's no real on-off.  Setting the color to 0,0,0 is off.  Setting it to non-0.0.0 is on.